docs/generate_includes.py

```diff
@@ -3,25 +3,48 @@
 
 """ Generate includes for documentation """
 
-import os
+import re
+import warnings
+from pathlib import Path
 
 import yaml
 
+STORAGE_FLAG_REGEXP = r"^/g/data/(?P<proj>[a-z]{1,2}[0-9]{1,2})/.*?$"
+
+
+def storage_includes() -> None:
+    here = Path(__file__).parent.absolute()
+    # here = os.path.abspath(os.path.dirname(__file__))
+
+    project_list = set()
+    for source_yaml in (here.parent / "config").glob("*.yaml"):
+        print(source_yaml)
```

Comment:
WIP?

Reply:
Ah that one wasn't, I figured it would be good to have a `stdout` list of used files in case someone sees something they think should(n't) be listed in the documentation.

Reply:
Makes sense